cmake/test_model_zoo.cmake

```diff
@@ -8,7 +8,7 @@ if(ENABLE_OV_ONNX_FRONTEND)
     # if requirements are not installed automatically, we need to checks whether they are here
     ov_check_pip_packages(REQUIREMENTS_FILE "${OpenVINO_SOURCE_DIR}/src/frontends/onnx/tests/requirements.txt"
                           RESULT_VAR onnx_FOUND
-                          WARNING_MESSAGE "ONNX frontend tests will be skipped"
+                          WARNING_MESSAGE "ONNX testing models weren't generated, some tests will fail due .onnx models not found"
```

Comment:
we can adjust message for PDPD and other places. Please, grep by `ov_check_pip_packages`

Reply:
Thanks! Done